test/extension/generalMobilePlatform.test.ts

```diff
@@ -29,5 +30,38 @@ suite("generalMobilePlatform", function () {
                 assert.equal(GeneralMobilePlatform.getOptFromRunArgs(args, "param4", false), undefined);
             });
         });
+
+        suite("getEnvArgument", function() {
+            let origEnv: any = {"test1": "origEnv", "test2": "origEnv", "test3": "origEnv"};
+            let env: any = {"test2": "env", "test3": "env", "test4": "env"};
+            let envForFile: string = "test3=envFile\ntest4=envFile\ntest5=envFile";
+            let envFile: string = ".env";
```

Comment:
Make sure to create this file not in the repository root, but in special temporary resources folder to avoid any side effects for this test.